torch_frame/nn/encoder/stype_encoder.py

```diff
@@ -819,7 +819,18 @@ def encode_forward(
                     for key in feat
                 })
             else:
-                x = self.model_dict[col_name](feat[:, i])
+                input_feat = feat[:, i]
+                if isinstance(input_feat, MultiEmbeddingTensor):
+                    input_feat = input_feat.values
```

Comment:
Can you add some comment why we take the values?

Reply:
Personally, I prefer to just pass the raw `MultiEmbeddingTensor` since that behavior is more predictable.